pkg/plugin/errors.go

```diff
@@ -14,6 +14,10 @@
 
 package plugin
 
-import "fmt"
+type invalidJustificationError string
```

Comment:
I think we want this to be:

```suggestion
type Error string
```

and then:

```go
const ErrInvalidJustification = Error("invalid justification")
```

That way you can define future errors like this:

```go
const ErrOtherThing = Error("other thing")
```

Reply:
Thanks! Updated!